packages/cli/__tests__/zosfiles/__system__/delete/uss/cli.files.delete.file.system.test.ts

```diff
@@ -123,13 +123,22 @@ describe("Delete File", () => {
             expect(response.status).toBe(0);
             expect(response.stdout.toString()).toMatchSnapshot();
         });
+
         it("should delete a file with response timeout", async () => {
             const response = runCliScript(__dirname + "/__scripts__/command/command_delete_file.sh",
                 TEST_ENVIRONMENT, [ussname, "--for-sure", "--responseTimeout 5"]);
             expect(response.stderr.toString()).toBe("");
             expect(response.status).toBe(0);
             expect(response.stdout.toString()).toMatchSnapshot();
         });
+
+        it("should delete a file with --ignore-not-found flag", async () => {
```

Comment:
Should this test have a second step to match the related test for data sets?
`// Now delete the USS file, verify no output in ignore-not-found mode`